code/api_crawler.py

```diff
@@ -79,7 +120,7 @@ def addNewLink(uri, response):
     if (uri.find('{')>-1):
         uri = expandTemplatedUri(uri, response)
 
-    if uri not in links:
+    if uri not in links and uri is not None:
```

Comment:
Om geen templated links mee te nemen waarvan wel een hostname is gezet (bijvoorbeeld voor de links naar de BAG API) heb ik deze check uitgebreid met:
`if uri not in links and uri is not None and not uri.startswith("http"):`